src/kitty-doom.h

```diff
@@ -42,6 +42,7 @@ typedef struct {
  */
 typedef struct input input_t;
 
+bool kitty_keyboard_protocol_supported(void);
```

Comment:
This is a one-time internal check. Should it really be public API? 

Consider making it static in input.c if not needed externally.